app/serializers/compliance_serializer.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+# JSON serializer parent class
+module ComplianceSerializer
+  extend ActiveSupport::Concern
+  include ::JSONAPI::Serializer
+
+  def initialize(resource, options = {})
+    options[:params] ||= {}
+    options[:params][:root_resource] = resource.class
+
+    super(resource, options)
+  end
+
+  class_methods do
+    def relationships_hash(record, _, _, _, params = {})
+      return nil if params[:root_resource] != record.class
```

Comment:
I'm not entirely sure about the `nil` here, see the JIRA issue for more...

Reply:
Empty Hash/JSON Object could be more appropriate.

This makes `relationships` empty in any case, even on top level items.

Reply:
That has been adjusted :wink: gonna change the `nil` to `{}` then...